modules/ui/rapid_internal_features_storage.js

```diff
@@ -0,0 +1,54 @@
+var memStorage = {}; 
+
+
+export function rapidInternalFeaturesStorage() {
+    // Cloned from context, but we need storage from within the actions
```

Comment:
This means that window.localStorage will have some fairly obvious keys in it that a user could enable/disable if they wanted to. Is this bad? Should these be obfuscated more?